lib/vmdb/console_methods.rb

```diff
@@ -28,6 +28,7 @@ def simulate_queue_worker(break_on_complete = false)
         else
           break_on_complete ? break : sleep(1.second)
```

Comment:
yes.

You said run 5 times, but the queue is empty. so `break`

you want it to run the remaining number of times?